metadata.json

```
@@ -15,41 +14,99 @@
     {
       "name": "puppetlabs/concat",
       "version_requirement": ">= 2.2.0 < 8.0.0"
+    },
+    {
+      "name": "puppet/systemd",
+      "version_requirement": ">= 3.6.0 < 4.0.0"
```

we released 3.7.0 today:
```suggestion
      "version_requirement": ">= 3.7.0 < 4.0.0"
```